docs/source/python/dataset.rst

```diff
@@ -731,6 +731,7 @@ to supply a visitor that will be called as each file is created:
 
     def file_visitor(written_file):
         print(f"path={written_file.path}")
+        print(f"size={written_file.size:,} bytes")
```

Comment:
This can be misleading because the comma is a decimal separator in some countries such as France, but perhaps that's just me :-). I would just use the default formatting. Or you can use `_` (which is recognized in Python int literals):
```suggestion
        print(f"size={written_file.size:_} bytes")
```

Reply:
Oh that's a good point. I'll just remove it. I with there was a localized version built-in to format strings.